docker/configs/php/xdebug.ini

```diff
@@ -12,7 +12,7 @@ xdebug.log_level = 1
 xdebug.discover_client_host=true
 xdebug.client_host=host.docker.internal
 
-# set this to autostart debugging e.g. if you cannot specify the trigger in the request
+# When you cannot specify a trigger, use "xdebug.start_with_request = yes" to autostart debugging
```

Comment:
Great suggestion. Maybe to make the impact even clearer we could extend that to:

```suggestion
# When you cannot specify a trigger, use "xdebug.start_with_request = yes" to autostart debugging for all requests
```